test/core/reporters/na.js

```diff
@@ -245,4 +245,25 @@ describe('reporters - na', function() {
 			done();
 		});
 	});
+	it('should add version information', function(done) {
+		axe.run(naOption, function(err, results) {
+			try {
```

Comment:
This `try/catch` seems very awkward. Can't we just do:

```js
axe.run(naOption, function (err, results) {
  if (err) {
    return done(err)
  }

  // Assertions go here
  done()
})
```

Reply:
The `try/catch` is for the assertions. A failed assertion throws an error but `done` is not called with it, so the unit test just hangs until timeout, then stops the test there without reporting what the error was.

Reply:
Ohh, wow, OK. That's something we should change. I wonder if updating `mocha` would solve the problem for us.

Obviously don't worry about this now.